src/config/settings.js

```diff
@@ -38,17 +38,34 @@ function load() {
   let settings;
   try {
     settings = JSON.parse(rawdata);
-  } catch(_) {
-    settings = {}
+  } catch (_) {
+    save(defaultSettings);
+    return defaultSettings;
   }
-  // clone settings because we check to see if the returned value is different
   let cfg = updateSettings(Object.assign({}, settings));
   if (!isDeepEqual(cfg, settings)) {
     save(cfg);
   }
   return cfg;
 }
 
+/**
+ * Verify that the loaded version of kubernetes, if specified, is in the current list of supported versions.  Throw an exception if not.
+ * @param{Object} settings
+ */
+
+function verifyLocalSettings(settings) {
+  const supportedVersions = require ('@/generated/versions.json')
```

Comment:
```suggestion
  const supportedVersions = require ('@/generated/versions.json');
```
I think we should get a lint error now at least? :)

Reply:
eslint added them on the post-rebase testing